aggregator_core/src/datastore/test_util.rs

```diff
@@ -261,12 +261,33 @@ impl Drop for EphemeralDatastore {
         // Make a best-effort attempt to delete the database created for this datastore. This may
         // fail if the container hosting the database server is deleted faster, which would make
         // this moot.
-        let pool = self.pool.clone();
         let db_name = self.db_name.clone();
-        spawn(async move {
-            if let Ok(conn) = pool.get().await {
-                let _ = conn.execute("DROP DATABASE $1", &[&db_name]).await;
-            }
+        let connection_string = self.db.connection_string("postgres");
+
+        // Since each unit test has its own Tokio runtime, we need to run `DROP DATABASE` in a
```

Comment:
Starting a new tokio runtime & connecting to the database seems kinda heavyweight for a cleanup-on-drop operation.

Could `EphemeralDatabase` maintain a single background thread/task (running on its own runtime, if necessary), which holds a connection open to the default `postgres` DB, and receives DB names to drop over an MPSC channel? (admittedly, this is getting kind of complicated for a cleanup operation.)